catalogue-service-micro/src/main/java/eu/nimble/service/catalogue/impl/CatalogueLineController.java

```diff
@@ -97,6 +98,9 @@ public ResponseEntity getCatalogueLineByHjid(@ApiParam(value = "Identifier of th
             produces = {"application/json"},
             method = RequestMethod.GET)
     public ResponseEntity getCatalogueLinesByHjids(@ApiParam(value = "Identifiers of the catalogue lines to be retrieved. (line.hjid)", required = true) @RequestParam(value = "ids") List<Long> hjids,
+                                                   @ApiParam(value = "Number of catalogue lines to be included in CataloguePaginationResponse ") @RequestParam(value = "limit",required = true) Integer limit,
```

Comment:
Since this service does not return CataloguePaginationResponse, ApiParam value is not correct.It could be something like that:@ApiParam(value = "Number of catalogue lines to be included in the result") @RequestParam(value = "limit",required = true) Integer limit,